### ISSUE TITLE
Binary for armhf fails with "-bash: ./docker-compose: No such file or directory"

### ISSUE THREAD

**Issue by alandrees** [docker/compose#7098]
## Description of the issue
Attemping to build on armhf yields a binary which doesn't seem to run on the host OS.

## Context information (for bug reports)
- Raspberry pi4
- Raspbian Buster
- Built using the Dockerfile, not the Dockerfile.armhf (although both seem to produce binaries which have similar issues)

**Output of `docker-compose version`**
```
$ docker-compose --version
-bash: /usr/local/bin/docker-compose: No such file or directory
```

**Output of `docker version`**
```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.2
 API version:       1.40
 Go version:        go1.12.8
 Git commit:        6a30dfc
 Built:             Thu Aug 29 06:18:36 2019
 OS/Arch:           linux/arm
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.2
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.8
  Git commit:       6a30dfc
  Built:            Thu Aug 29 06:12:41 2019
  OS/Arch:          linux/arm
  Experimental:     false
 containerd:
  Version:          1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

```

**Output of `docker-compose config`**
(Make sure to add the relevant `-f` and other flags)
```
$ docker-compose config
-bash: /usr/local/bin/docker-compose: No such file or directory
```

## Steps to reproduce the issue

1.  docker build -f Dockerfile .
2. docker cp container:/usr/local/bin/docker-compose ./
3. run the copied version of docker-compose `./docker-compose`

### Observed result
-bash: /usr/local/bin/docker-compose: No such file or directory
### Expected result
Fully functioning docker-compose

### Stacktrace / full error message

```
(paste here)
```

## Additional information

OS version / distribution, `docker-compose` install method, etc.

**Comment by ajithreddygeth** [comment]
@alandrees `docker-compose` package need to install separately. it won't come with standard docker package 

install docker-compose using pip 
`sudo pip install docker-compose`

when you run `docker-compose` at first you will get the error the fix id 
`cp -r /usr/local/lib/python2.7/dist-packages/backports/ssl_match_hostname/ /usr/lib/python2.7/dist-packages/backports`

**Comment by alandrees** [comment]
I cloned the docker-compose repository and built the Docker image using that.

I then copied the binary `/usr/local/bin/docker-compose` to my host, and proceeded to get that error.

I was hoping to not have to install all the build dependencies on my production machines, is this just not possible for arm?

**Comment by hraftery** [comment]
Coincidentally I've been trying to do this for a few days now. I failed very many ways, including this one, and have finally succeeded. My secret trick was to stop reading the documentation and just start running code. Turns out that running `./script/build/linux` (after running `sudo python3 setup.py develop`) does it all for you, in just the right way. Out pops this 14MB executable:

```
./dist/docker-compose-Linux-armv7l --version
docker-compose version 1.26.0dev, build e9220f45
```
